src/pages/workspaces/[workspaceId]/settings.tsx

```diff
@@ -115,6 +147,21 @@ const WorkspaceSettingsPage: NextPageWithLayout = (props: Props) => {
             <div></div>
           </Tabs.Tab>
         </Tabs>
+        <DeleteWorkspaceDialog
+          workspace={workspace}
+          open={isDeleteDialogOpen}
+          onClose={() => {
+            setIsDeleteDialogOpen(false);
+          }}
+        />
+        <InviteMemberDialog
+          open={isNewMemberDialogOpen}
+          onClose={() => {
+            setIsNewMemberDialogOpen(false);
+            refetch({ id: workspace.id, page: 1, perPage: 5 });
+          }}
+          workspace={workspace}
```

Comment:
You should not do this here. The `InviteMembersDialog` should use the `useCacheKey` that would be catch in the `WorkspaceMembers` component